parser/protobuf_usp_test.go

```diff
@@ -0,0 +1,93 @@
+package parser_test
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/telenornms/skogul/gen/usp"
+	"github.com/telenornms/skogul/parser"
+)
+
+func readFile(file string, t *testing.T) []byte {
+	t.Helper()
+	b := make([]byte, 9000)
+	f, err := os.Open(file)
+	if err != nil {
+		t.Fatalf("unable to open protobuf packet file: %v", err)
+	}
+	defer f.Close()
+	n, err := f.Read(b)
+	if err != nil {
+		t.Fatalf("unable to read protobuf packet file: %v", err)
+	}
+	if n == 0 {
+		t.Fatalf("read 0 bytes from protobuf packet file....")
+	}
+	return b[0:n]
+}
+
+func TestParseFile(t *testing.T) {
```

Comment:
Keep in mind that the scope is for all parser tests, so most exported names should probably have USP in them to avoid future conflicts. (Not that we have been all too strict about this thus far). E.g.: TestUSPParseFile. It also makes it easier when running tests : go test -v -run .*USP.*